UnityProject/Assets/Scripts/UI/Objects/Medical/Cloning/GUI_CloningItemList.cs

```diff
@@ -1,53 +0,0 @@
-﻿using Objects.Medical;
-using UI.Core.NetUI;
-
-namespace UI.Objects.Medical.Cloning
-{
-	public class GUI_CloningItemList : NetUIDynamicList
-	{
-		public void AddItems(int count)
-		{
-			for (int i = 0; i < count; i++)
-			{
-				AddItem();
-			}
-		}
-
-		public bool AddItem()
```

Comment:
oh so, it no longer saves it in machine?

Reply:
It no longer does.

Reply:
yeup

Reply:
Think it might be good for the cloner to store like the most recent scan in like a buffer.
If you want to store any more than that you need to file your papers.

Reply:
Unless it already does that then yeah

Reply:
> Think it might be good for the cloner to store like the most recent scan in like a buffer. If you want to store any more than that you need to file your papers.

The current implementation is better for usability, as It's pretty much straight forward and more interactive.